portfolio/src/main/webapp/script.js

```diff
@@ -285,6 +285,10 @@ function createDivElement(comment, isBlogComment) {
   pElementEmoji.innerText = comment.emoji;
   divElement.appendChild(pElementEmoji);
 
+  const imgElement = document.createElement('a');
+  imgElement.innerHTML = "<a href=\"" + comment.image + "\"><img src=\"" + comment.image + "\" /></a>"
```

Comment:
You can set attributes in the element directly without concatenating HTML.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement